simulationworkflowschema/molecular_dynamics.py

```diff
@@ -384,6 +392,156 @@ class BarostatParameters(ArchiveSection):
     )
 
 
+class Lambdas(ArchiveSection):
```

Comment:
Yes, I definitely will do this, but in a different PR since this will involve some significant changes.

What is the difference between `BaseProperty` and `Property`? Just that we would specify the type with Enums? I feel like I am missing something.

For Properties, I did create a hierarchy to pass common parameters, I have `EnsembleProperty`, `CorrelationFunction`, and `TrajectoryProperty`, and they also inherit from `Property`, but again I feel that I am missing something here. Perhaps it's easiest to briefly meet later to clarify? I will message you on discord.